GDJS/Runtime/runtimescene.js

```diff
@@ -521,6 +521,14 @@ gdjs.RuntimeScene.prototype.updateObjectsForces = function() {
  * @param obj The object to be added.
  */
 gdjs.RuntimeScene.prototype.addObject = function(obj) {
+    if ( !this._objects.containsKey(obj.name) ) {
+        this._objects.put(obj.name, obj.getOriginalData());
```

Comment:
Would it be possible to move this into a "registerObject" (or `loadObject`, still not sure) method? Passing the name, constructor, and objectData.

I'm still unsure about what's the ideal final form of this, but with a separate method we could: 
* avoid code duplication (this code and remove `loadObject` in `loadFromScene` in favor of this new method).
* could make clearer that there is a step of registering/loading needed before handling objects.